### ISSUE TITLE
[UI] L'illustration d'un article est trop grande

### ISSUE THREAD

**Issue by Nozalys** [NextINpact/Next#32]
Bonjour,
J'ai compris que vous vouliez lui donner plus d'importance, mais pour moi l'illustration d'un article dans la page de lecture de l'article prend beaucoup trop de place et n'apporte rien sur le fond. Dans la page d’accueil elle participe à l'envie (ou pas) de cliquer, et c'est important.
Mais quand j'ai cliqué, j'aime bien pouvoir commencer à lire sans avoir à scroller sur l'équivalent de + d'un écran.

C'est d'autant plus pénible si on ouvre plusieurs articles dans différents onglets pour commencer à les parcourir.

Éventuellement si vous souhaitez vraiment laisser ce comportement par défaut, pourquoi pas envisager un paramètre du profile utilisateur.

**Comment by Glandos** [comment]
Pour comparaison, la version mobile est plutôt raisonnable :
![image](https://github.com/NextINpact/Next/assets/297578/00adc347-9b3e-4127-9d1e-1de642b00227)

Par contre, la version bureau prend effectivement toute la place :
![image](https://github.com/NextINpact/Next/assets/297578/c9620d83-c6ce-47e9-a681-b9e458f778b8)

C'est un souci que j'ai déjà essayé de résoudre par le passé, mais sans solution parfaite :
- Mettre l'image plus petite en hauteur. La largeur est réduite, ça fait moche.
- Mettre l'image en dessous du chapeau. Ça fait scroller aussi, mais du contenu est quand même lisible.
- Mettre l'image en encart, façon enluminure. Ça ne fait pas hommage à la qualité de l'illustration, et c'est souvent mieux en format carré, ce qui n'est pas le format des illustrations.

En tout cas, mettre le choix entre au moins deux options, oui, c'est intéressant :)

**Comment by eglyn** [comment]
Je suis d'accord, l'image est bien trop grande, sachant qu'elle n'apporte rien à l'article, la réduire, et passer le titre au dessus serait peut-être une solution, comme sur le site actuel.

**Comment by Nozalys** [comment]
@eglyn "_Je rajoute que dans le site actuel, l'image est un support pour la catégorie et l'auteur ainsi que des boutons de signalement, etc... Ce qui rend l'intégration de l'image très bien faite. On n'a pas juste une image parce qu'il faut juste une image pour meubler._"

Je suis parfaitement d'accord. L'intégration actuelle apporte une valeur ajoutée contrairement à une simple illustration d'article.

**Comment by nathanael-h** [comment]
Bonjour, 
Pour compléter cette issue, j'ai remarqué que des images de grandes tailles sont chargées pour afficher des miniatures. Cela ralenti considérablement le chargement de la page. 
Voici un exemple : 
![image](https://github.com/NextINpact/Next/assets/7300309/04bee01d-3c6b-4332-bee6-9a7bea22ecdf)

Si vous pouvez prévoir des miniatures ça serait top (et ça vous économiserait un peu de bande passante, certes au détriment d'un peu plus de stockage).

**Comment by AnaelMobilia** [comment]
@nathanael-h C'est #116 ;-)

L'API WP JSON fournit désormais des médias de taille redimensionnées. C'est donc dans les tuyaux :)
![image](https://github.com/NextINpact/Next/assets/6187125/a7f25aa7-f94b-4f13-a6dd-ea0823614f5e)

**Comment by dragonis41** [comment]
Je laisse cette issue ouverte en attendant, mais pour avoir un petit update sur l'avancement  :
- On a réduit la taille de beaucoup d'images
- On travaille sur un plugin qui va permettre de réduire automatiquement la taille de toutes les images quand les journalistes vont les uploader
- Il y a déjà quelques crédits sur les nouveaux articles, on regarde pour importer les anciens crédits et les ajouter à la volée sur les vieux articles

**Comment by dragonis41** [comment]
Petit update :

* Toutes les images ont été réduites
* Il y a du lazyloading pour réduire la charge
* On n'a toujours pas importé les anciens crédits car c'est assez fastidieux à faire mais c'est prévu
